### PR TITLE
Add terms and conditions customization through active admin

### PR DESCRIPTION
**Pull request by beasrodrigues24**
Why: 
* We want the admin to be able to customize the terms and conditions link, used in the privacy policy page

How:
* By creating a model and migration for the terms and conditions, containing a link
* By generating the active admin resource and removing the create possibility (we want the admin to just re-edit the same one)
* By fixing the front-end to fetch this link from the database

### CODE REVIEW

File: app/controllers/terms_and_conditions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Controller that manages terms and conditions
+class TermsAndConditionsController < ApiController
+  def index
+    render json: TermsAndCondition.find(1).slice(:link)
```
Comment (zediogoviana) [inline app/controllers/terms_and_conditions_controller.rb:6]: Why is the index hardcoded to `find(1)`?

Comment (beasrodrigues24) [reply app/controllers/terms_and_conditions_controller.rb:6]: So we want there to only be one of the Terms and Conditions. I didn't know how to do that limitation any other way :sweat_smile:

Comment (zediogoviana) [reply app/controllers/terms_and_conditions_controller.rb:6]: Then why not use `show` or some custom naming? I think it'd be more explicit given that you'll only receive one object, and not "all".

Comment (beasrodrigues24) [reply app/controllers/terms_and_conditions_controller.rb:6]: Okay, I'll change it to show :+1:

Comment (beasrodrigues24) [reply app/controllers/terms_and_conditions_controller.rb:10]: Done!!

**Review by zediogoviana — APPROVED** [review]

**Pull request merged**